v2.0/import.md

```diff
@@ -357,12 +256,19 @@ WITH
 )
 CSV DATA ('azure://acme-co/customer-import-data.tsc?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co')
 WITH
-	temp = 'azure://acme-co/temp/?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co',
 	nullif = ''
 ;
 ~~~
 
+## Known Limitation
+
+`IMPORT` can sometimes fail with a "context canceled" error, or can restart itself many times without ever finishing. If this is happening, it is likely due to a high amount of disk contention. This can be mitigated by setting the `kv.bulk_io_write.max_rate` setting to a value below your max disk write speed. For example, to set it to 10MB/s, execute: 
```

Comment:
nit: `setting` > `cluster setting`. Also, please make cluster setting a link to `cluster-settings.html`.

Reply:
Done.